libs/content-watcher-lib/src/crawler/crawler.service.ts

```diff
@@ -79,7 +82,7 @@ export class CrawlerService extends BaseConsumer {
           this.logger.debug(`Found ${messages.length} messages for block ${blockNumber}`);
         }
         // eslint-disable-next-line no-await-in-loop
-        await this.chainEventService.queueIPFSJobs(messages, this.ipfsQueue, clientReferenceId);
+        await ChainEventProcessorService.queueIPFSJobs(messages, this.ipfsQueue, clientReferenceId);
```

Comment:
Double check this change.

Reply:
It's also in one or two other places

Reply:
Lint was complaining about `queueIPFSJobs` not referencing `this`; since it doesn't, to resolve the lint error I made it a static method.